src/install/memcached/linux/install-yum.mdx

```diff
@@ -0,0 +1,14 @@
+---
+componentType: default
+headingText: Download using Yum
+---
+
+1. From the command line, run these commands:
```

Comment:
```suggestion
From the command line, run these commands:
```

Reply:
i think i made suggestions on another one of these parallel components to unspool the directions a bit more. if you go w/ em, don't forget to add those suggestions to these other ones